examples/grid_example/public/types.ts

```diff
@@ -36,7 +36,7 @@ export interface MockedDashboardPanelMap {
 }
 
 export interface MockedDashboardRowMap {
```

Comment:
what is `Row` here?

Reply:
Oops! Forgot the `row -> section` rename here. Hopefully dbdfa8f878b9ef2628c98a85f3aea0b62a429d41 makes it clearer 🙈